docs/learn/documentation/versioned/jobs/configuration-table.html

```diff
@@ -822,6 +828,40 @@
                 </tr>
 
                 <tr>
+                    <th colspan="3" class="section" id="streams"><a href="../container/streams.html">Streams</a></th>
+                </tr>
+
+                <tr>
+                    <td class="property" id="streams-system">streams.<span class="stream">stream-id</span>.samza.system</td>
+                    <td class="default"></td>
+                    <td class="description">
+                        The <span class="system">system-name</span> of the system on which this stream will be accessed.
+                        This property binds the stream to one of the systems defined with the property
+                        systems.<span class="system">system-name</span>.samza.factory. <br>
+                        If this property isn't specified, it is inherited from job.default.system.
+                    </td>
+                </tr>
+
+                <tr>
+                    <td class="property" id="streams-physical-name">streams.<span class="stream">stream-id</span>.samza.physical.name</td>
```

Comment:
> Co-existence of systems.<xyz>.streams.<abc>. and streams.<abcId>.system... is confusing. We might want to mark the whole section of systems.<xyz>.streams.<abc> as deprecated, and re-write w/ the new set of configuration to avoid this confusion.

That's not a bad idea, but I likely won't be able to get that done and reviewed before EOD Monday. So some options:
1. I checkin this patch to the integration branch as is and file a JIRA to do the rewriter before merging into master. This could be done by someone else if I'm not back in time, or I could take it up when I'm back.
2. Start working on it and see how far I get by EOD Monday. Update this PR and have someone else take over if need be.